README.md

```diff
@@ -130,52 +130,46 @@ git clone https://github.com/pytti-tools/pytti-notebook
 
       ```
       # Download this codebase
-      git clone https://github.com/pytti-tools/pytti-core
-
-      # Download research code from reference repositories
-      git clone https://github.com/openai/CLIP
-      git clone https://github.com/CompVis/taming-transformers
-
-      # Download research code modified to integrate better with PYTTI
-      git clone https://github.com/pytti-tools/AdaBins
-      git clone https://github.com/pytti-tools/GMA
+      git clone --recurse-submodules https://github.com/pytti-tools/pytti-core
       ```
 
     The end result should be a separate folder for each of pytti and the downloaded models. You should now have a folder structure that looks something like this:
          
             ├── pytti-notebook
-            │   ├── AdaBins
-            │   ├── CLIP
             │   ├── config
```

Comment:
double config/...

also, make sure we're referncing this config location and not the site-packages config when we do the CLI thing.